frontend/cypress/integration/editor/register.spec.js

```diff
@@ -0,0 +1,45 @@
+import { v4 as uuid } from 'uuid'
+  
+describe('When visiting the register page, as a user', () => {
+    
+    beforeEach(() => {
+      cy.visit(Cypress.env('HOST') + '/register')
+    })
+  
+    it('I see an error message when I try to register with only username', () => {
+      cy.get('#username').type('testuser')  
+      cy.get('form button').click()
+      cy.get('#email-helper-text').should('have.css', 'color', 'rgb(176, 0, 32)')
+      cy.get('#password-helper-text').should('have.css', 'color', 'rgb(176, 0, 32)')
```

Comment:
would it be better to test that the error message is the right error rather than check the color? or check both color and message

Reply:
These tests need to be revised when front end validation is merged.